### PR TITLE
Add tempest job based on EDPM multinode

### PR DESCRIPTION
**Pull request by rlandy**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: zuul.d/tempest_multinode.yaml [hunk 1]
```diff
@@ -72,3 +72,31 @@
...
       - ^scenarios/centos-9/ceph_backends.yml
       - ^zuul.d/tempest_multinode.yaml
+
+- job:
+    name: periodic-podified-multinode-edpm-deployment-antelope-ocp-crc-cs9-tempest-api
+    parent: periodic-podified-multinode-edpm-deployment-antelope-ocp-crc-cs9
+    vars:
+      cifmw_test_operator_tempest_include_list: |
+        ^tempest.api.identity.*.v3
+        # ^tempest.api.volume
```
Comment (marios) [inline zuul.d/tempest_multinode.yaml:82]: is that one commented on purpose? if it is waiting for something please add a comment (e.g. # TODO(rlandy) enable api.volume after xyz merged

File: zuul.d/tempest_multinode.yaml [hunk 2]
```diff
@@ -72,3 +72,31 @@
       - ^scenarios/centos-9/ci.yml
       - ^scenarios/centos-9/ceph_backends.yml
       - ^zuul.d/tempest_multinode.yaml
+
```
Comment (raukadah) [inline zuul.d/tempest_multinode.yaml:75]: We define periodic jobs here: https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/edpm_periodic.yaml Can we move this job definition in the `uul.d/edpm_periodic.yaml ` file?

File: zuul.d/tempest_multinode.yaml [hunk 3]
```diff
@@ -63,11 +63,47 @@
...
 
+- job:
+    name: podified-multinode-edpm-deployment-crc-tempest
+    parent: podified-multinode-edpm-deployment-crc
+    vars:
+      cifmw_extras:
+        - '@scenarios/centos-9/ci.yml'
+        - '@scenarios/centos-9/multinode-ci.yml'
+        - '@scenarios/centos-9/ceph_backends.yml'
+      cifmw_tempest_tempestconf_config:
```
Comment (viroel) [inline zuul.d/tempest_multinode.yaml:74]: I think that this one will override the cifmw_tempest_tempestconf_config definition set in the scenario file

**Comment by son-vyas** [comment]
Thanks for adding the job but i see most of tempest tests have skipped - https://logserver.rdoproject.org/52/52652/39/check/podified-multinode-edpm-deployment-crc-tempest/709a863/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

**Review by marios — COMMENTED** [review]
+1 for now i guess we will discuss in today's reviews

File: scenarios/centos-9/ceph_backends.yml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-cifmw_control_plane_ceph_backend_include_vars: "{{ cifmw_basedir }}/artifacts/pre_deploy_ceph_deploy.yml"
+cifmw_control_plane_ceph_backend_include_vars: "{{ cifmw_basedir }}/artifacts/pre_deploy_61_ceph_deploy.yml"
```
Comment (pablintino) [inline scenarios/centos-9/ceph_backends.yml:2]: It looks like a fragile variable.

**Review by viroel — COMMENTED** [review]
looks good

**Comment by rlandy** [comment]
latest testproject results: https://logserver.rdoproject.org/52/52652/44/check/podified-multinode-edpm-deployment-crc-tempest/21ef745/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

**Comment by viroel** [comment]
/approve

test results looks good

**Pull request merged**